Snakefile

```diff
@@ -0,0 +1,496 @@
+        GS.remote(f"{config['SLAPEnrich']['outputBucket']}/slapenrich-{timeStamp}.json.gz"),
+        GS.remote(f"{config['Gene2Phenotype']['outputBucket']}/gene2phenotype-{timeStamp}.json.gz"),
+        GS.remote(f"{config['CRISPR']['outputBucket']}/crispr-{timeStamp}.json.gz"),
+        GS.remote(f"{config['SysBio']['outputBucket']}/sysbio-{timeStamp}.json.gz"),
+        GS.remote(f"{config['Phenodigm']['outputBucket']}/phenodigm-{timeStamp}.json.gz")
+
+# --- Auxiliary Rules --- #
+## help               : prints help comments for Snakefile
+rule help:
+    input: "Snakefile"
+    shell:
+        "sed -n 's/^##//p' {input}"
+
+## clean               : prints help comments for Snakefile
+rule clean:
+    shell:
+        "rm -rf tmp"
+
+# --- Data sources parsers --- #
+## clingen          : processes the Gene Validity Curations table from ClinGen
+rule clingen:
+    input:
+        f"tmp/ClinGen-Gene-Disease-Summary-{timeStamp}.csv"
+    output:
+        evidenceFile=GS.remote(f"{config['ClinGen']['outputBucket']}/ClinGen-{timeStamp}.json.gz"),
+        unmappedDiseases=f"tmp/unmappedDiseases/clingen_unmapped_diseases-{timeStamp}.lst"
+    conda:
+        'envs/conda-env.yml'
+    log:
+        GS.remote(logFile)
+    shell:
+        """
+        python modules/ClinGen.py \
+        --input_file {input} \
+        --output_file {output.evidenceFile} \
+        --unmapped_diseases_file {output.unmappedDiseases}
+        """
+
+## geneticsPortal           : processes lead variants from the Open Targets Genetics portal on a Dataproc cluster
+rule geneticsPortal:
+    shell:
+        """
+        gcloud beta dataproc clusters create \
+            snakemake-cluster-l2g-data \
+            --image-version=1.4 \
+            --properties=spark:spark.debug.maxToStringFields=100,spark:spark.executor.cores=31,spark:spark.executor.instances=1 \
+            --master-machine-type=n1-standard-32 \
+            --master-boot-disk-size=1TB \
+            --zone=europe-west1-d \
+            --single-node \
+            --max-idle=10m \
+            --region=europe-west1 \
+            --project=open-targets-genetics
+        
+        gcloud dataproc jobs submit pyspark \
+            --cluster=snakemake-cluster-l2g-data \
+            --project=open-targets-genetics \
+            --region=europe-west1 \
+            modules/GeneticsPortal.py -- \
+            --locus2gene gs://genetics-portal-data/l2g/200127 \
+            --toploci gs://genetics-portal-data/v2d/200207/toploci.parquet \
+            --study gs://genetics-portal-data/v2d/200207/studies.parquet \
+            --threshold 0.05 \
+            --variantIndex gs://genetics-portal-data/variant-annotation/190129/variant-annotation.parquet  \
+            --ecoCodes gs://genetics-portal-data/lut/vep_consequences.tsv \
+            --outputFile gs://genetics-portal-analysis/l2g-platform-export/data/genetics_portal_evidence.json.gz
+
+        gcloud dataproc clusters delete \
```

Comment:
The delete is not necessary: the cluster is deleted after 10 minutes idle time `--max-idle=10m `, but can stay.

Reply:
It was just for peace of mind 😬

Reply:
Addressed in `9725f11`